deploy/templates/serviceaccount.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     {{- if .Values.serviceAccount.annotations }}
     {{ toYaml .Values.serviceAccount.annotations . | nindent 4 }}
     {{- end }}
-    {{- if eq .Values.workloadIdentity.enabled true }}
+    {{- if and (.Values.workloadIdentity.enabled) (not .Values.workloadIdentity.disableGlobalServiceAccount) }}
```

Comment:
will this will this disable the service account? Looks like it only stops annotating with client id

Reply:
We can't actually get rid of this service account since it bind with clusterRoles, we still need it for accessing resources. Remove this annotation can make the service account lose the ability to use the workload identity since the Service Account Token Volume Projection can't work won't work without this annotation

Reply:
got it ,thanks